plugins/modules/jira.py

```diff
@@ -511,7 +519,7 @@ class JIRA(StateModuleHelper):
             ('operation', 'comment', ['issue', 'comment']),
             ('operation', 'workflow', ['issue', 'comment']),
             ('operation', 'fetch', ['issue']),
-            ('operation', 'transition', ['issue', 'status']),
+            ('operation', 'transition', ['issue', 'status', 'status_id']),
```

Comment:
This requires that you specify both `status` *and* `status_id` when `operation=transition`. You have to split it up as follows:
```suggestion
            ('operation', 'transition', ['issue']),
            ('operation', 'transition', ['status', 'status_id'], True),
```
This says that if `operation=transition`, then:
1) `issue` is required;
2) one of `status` and `status_id` is required.

Reply:
Ah okay, thanks for explaining that